examples/README.md

```diff
@@ -0,0 +1,47 @@
+# Celerity Example Projects
+
+These example projects are a useful starting point for developing your own Celerity
+application. All examples will also be built automatically in-tree with the runtime
+when the `CELERITY_BUILD_EXAMPLES` CMake option is set (true by default).
+
+## Setup
+
+Begin by copying an example project folder that best fits your use case,
+while resolving symlinks:
```

Comment:
I'd be a bit worried about the symlink part on some target platforms, but it's fine for now.